app/blockchain/fabric-samples/ci/scripts/run-k8s-test-network-basic.sh

```
@@ -0,0 +1,118 @@
+export CLIENT_LANGUAGE=${CLIENT_LANGUAGE:-typescript}
+export CHAINCODE_LANGUAGE=${CHAINCODE_LANGUAGE:-java}
+export TEST_NETWORK_CHAINCODE_BUILDER=${CHAINCODE_BUILDER:-ccaas}
+
+# Fabric version
+export TEST_NETWORK_FABRIC_VERSION=${FABRIC_VERSION:-}
+
+# Orderer parameters
+export TEST_NETWORK_ORDERER_TYPE=${ORDERER_TYPE:-raft}
+
+# test-network-k8s parameters
+export TEST_TAG=$(git describe)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)